Bonfire-Arguments-Optional.md

```diff
@@ -115,7 +115,40 @@ function add() {
     }
 }
 ```
+
+## Solution #3:
+
+```js
+// This is a more generic solution. 
+// It will return the sum of unlimited number of arguemnts if they are numbers 
+function add(b) {
+    // check if all passed arguments are numbers. 

```

Comment:
Don't you think it's better to move this comment one line below before `// return undefined if even one argument is not a number `. Otherwise I understand it like this: `for (var key in arguments) {` checks if all passed arguments are numbers.